src/validators/plugins/index.js

```diff
@@ -23,5 +23,5 @@ function validatePlugins(plugins) {
 // plugins can either be a function which has prototype.apply defined,
 // or a function that returns an object with apply property
 export function isPlugin(plugin) {
-  return !!plugin.apply || typeof plugin === 'function'
+  return !!plugin.apply || friendlyTypeOf(plugin) === 'function'
```

Comment:
I didn't get it, how is it different from [`const type = friendlyTypeOf(value)`](https://github.com/kentcdodds/webpack-validator/blob/master/src/validators/cache/index.js#L9)

May be, I don't understand, when to use `friendlyTypeOf` and `typeof`, could you enlighten me?